hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/node/NodeManager.java

```diff
@@ -199,4 +220,10 @@ void processNodeReport(DatanodeDetails datanodeDetails,
    * @return the given datanode, or null if not found
    */
   DatanodeDetails getNodeByAddress(String address);
+
+  /**
+   * Get cluster map as in network topology for this node manager.
+   * @return cluster map
+   */
+  NetworkTopology getClusterMap();
```

Comment:
Suggest to change the name to getClusterNetworkTopologyMap.

Reply:
Updated.